CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ---
 
+## Master
+
+### Bug Fixes
+
+* Fix code which checks is font already registered.
```

Comment:
typo: "if font"

Also, please add two spaces after that ending period. That's required by MarkDown syntax to render the next line separately properly (see other entries for inspiration)